pkg/sentry/kernel/fd_table.go

```diff
@@ -189,8 +188,10 @@ func (f *FDTable) DecRef(ctx context.Context) {
 func (f *FDTable) forEach(ctx context.Context, fn func(fd int32, file *fs.File, fileVFS2 *vfs.FileDescription, flags FDFlags)) {
```

Comment:
There now needs to be a lock/unlock in the call sites in saveDescriptorTable() and String()